device/accelerometer/index.ts

```diff
@@ -1,6 +1,6 @@
-import { AccelerometerBase } from './accelerometer';
+import { IAccelerometer } from './accelerometer';
 
-const Accelerometer: typeof AccelerometerBase = require(`./accelerometer.${Device.deviceOS.toLowerCase()}`).default;
-type Accelerometer = AccelerometerBase;
+const Accelerometer: IAccelerometer = require(`./accelerometer.${Device.deviceOS.toLowerCase()}`).default;
```

Comment:
`const Accelerometer: IAccelerometer` bu kullanim yanlis
`const Accelerometer: ConstructorOf<IAccelerometer, Partial<IAccelerometer>>` olmasi lazim

Reply:
we talked and decided with @cenkce its okay like this.